### PR TITLE
Updated readme for query interface

### PR DESCRIPTION
**Pull request by g3d**

### CODE REVIEW

**Comment by AlfonsoUceda** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -413,6 +413,29 @@ In the example above, we reuse the adapter because the target tables (`people` a
...
 An object that implements an interface for querying the database.
 This interface may vary, according to the adapter's specifications.
+
+Here is common interface for existing class:
+
+  * `.all`   - Resolves the query by fetching records from the database and translating them into entities
+  * `.where`, `.and`  - Adds a condition that behaves like SQL `WHERE`
+  * `.or` - Adds a condition that behaves like SQL `OR`ction
+  * `.exclude`, `.not`  - Logical negation of a #where condition
+  * `.select` - Select only the specified columns
```
Comment (joneslee85) [inline README.md]: s/Select/Selects ?

File: README.md [hunk 2]
```diff
@@ -413,6 +413,29 @@ In the example above, we reuse the adapter because the target tables (`people` a
 
 An object that implements an interface for querying the database.
 This interface may vary, according to the adapter's specifications.
+
+Here is common interface for existing class:
+
+  * `.all`   - Resolves the query by fetching records from the database and translating them into entities
+  * `.where`, `.and`  - Adds a condition that behaves like SQL `WHERE`
+  * `.or` - Adds a condition that behaves like SQL `OR`ction
```
Comment (joneslee85) [inline README.md]: s/ction/action

File: README.md [hunk 3]
```diff
@@ -413,6 +413,29 @@ In the example above, we reuse the adapter because the target tables (`people` a
 
 An object that implements an interface for querying the database.
 This interface may vary, according to the adapter's specifications.
+
+Here is common interface for existing class:
+
+  * `.all`   - Resolves the query by fetching records from the database and translating them into entities
```
Comment (joneslee85) [inline README.md]: s/   -/ -

File: README.md [hunk 4]
```diff
@@ -413,6 +413,29 @@ In the example above, we reuse the adapter because the target tables (`people` a
...
+  * `.where`, `.and`  - Adds a condition that behaves like SQL `WHERE`
```
Comment (joneslee85) [inline README.md]: s/ -/ -

File: README.md [hunk 5]
```diff
@@ -413,6 +413,29 @@ In the example above, we reuse the adapter because the target tables (`people` a
...
+  * `.or` - Adds a condition that behaves like SQL `OR`ction
+  * `.exclude`, `.not`  - Logical negation of a #where condition
```
Comment (joneslee85) [inline README.md]: s/ -/ -

**Comment by jodosha** [comment]
@g3d Thank you. Merging :+1:

**Pull request merged**